core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-container-code/dot-container-code.component.ts

```diff
@@ -140,7 +182,10 @@ export class DotContentEditorComponent implements OnInit {
                             })
                         );
 
-                        this.updateActiveTabIndex(this.getcontainerStructures.length);
```

Comment:
did you try doing this on `ngAfterViewInit()` ?

Reply:
it doesn't work in this case.

Reply:
What about: https://stackoverflow.com/questions/38709923/why-is-requestanimationframe-better-than-setinterval-or-settimeout maybe?

Reply:
updated